apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -453,74 +453,81 @@ function handleUpdateImageBlocks( calypsoPort ) {
  * @param {MessagePort} calypsoPort Port used for communication with parent frame.
  */
 function handlePreview( calypsoPort ) {
-	$( '#editor' ).on( 'click', '.editor-post-preview', ( e ) => {
-		e.preventDefault();
-		e.stopPropagation();
+	document.getElementById( 'editor' ).addEventListener(
+		'click',
+		( e ) => {
+			if ( ! e.target.classList.contains( 'editor-post-preview' ) ) {
```

Comment:
This condition is unreliable. It works only if the `<a>` element contains only a text node and nothing else. Like `<a>Preview</a>`. As soon as it contains an icon, like `<a><svg />Preview</a>` or something like `<a>Preview <b>This</b></a>`, the `event.target` can be the `svg` or `b` element, not the `a.editor-post-preview` one.

A better condition is:
```js
if ( ! e.target.closest( '.editor-post-preview' ) )
```
which traverses up the DOM tree and tries to find a matching parent element.

Reply:
Ooo, good spot. Do you know if we ship [the polyfill for closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#polyfill) as part of `wpcom-block-editor`? I think we're still supporting IE11